src/Helpers/URLHelper.php

```diff
@@ -38,4 +36,41 @@ public static function join(string $baseUrl, string $endpoint): string
 
         return $baseEndpoint . $endpoint;
     }
+
+    /**
+     * Check if the url pattern matches the value
+     *
+     * @param string $pattern
+     * @param string $value
+     * @return bool
+     */
+    public static function is(string $pattern, string $value): bool
+    {
+        if ($pattern === $value) {
+            return true;
+        }
+
+        $pattern = preg_quote($pattern, '#');
+        $pattern = str_replace('\*', '.*', $pattern);
+
+        if (preg_match('#^' . $pattern . '\z#u', $value) === 1) {
+            return true;
+        }
+
+        return false;
```

Comment:
How come we're not using the same code as Illuminate/support Str here? https://github.com/illuminate/support/blob/master/Str.php#L357